Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -223,6 +223,21 @@ public class ModelGroup {
       }
       else Debug.LogWarning("A group matching that name does not exisit in the modelPool");
     }
+    public void AddNewGroup(string groupName, IHandModel leftModel, IHandModel rightModel) {
+      ModelGroup newGroup = new ModelGroup();
+      newGroup.LeftModel = leftModel;
+      newGroup.RightModel = rightModel;
+      newGroup.GroupName = groupName;
+      newGroup.CanDuplicate = false;
+      newGroup.IsEnabled = true;
+      ModelPool.Add(newGroup);
+    }
+    public void RemoveGroup(string groupName) {
```

Comment:
This method will only remove one group if multiple groups exist with the same name.